testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -446,6 +454,61 @@ def test_stringio_outofrange(self, universe3):
             with mda.coordinates.PDB.PDBWriter(outstring) as writer:
                 writer.write(u.atoms)
 
+    def test_hetatm_written(self, universe4, tmpdir, outfile):
+        """
+        Checks that HETATM record types are written.
+        """
+
+        u = universe4
+        u_hetatms = u.select_atoms("resname ETA and record_type HETATM")
+        assert_equal(len(u_hetatms), 8)
+
+        u.atoms.write(outfile)
+        written = mda.Universe(outfile)
+        written_atoms = written.select_atoms("resname ETA and "
+                                             "record_type HETATM")
+
+        assert len(u_hetatms) == len(written_atoms), "mismatched HETATM number"
+        assert_almost_equal(u_hetatms.atoms.positions, written_atoms.atoms.positions)
```

Comment:
PEP8 > 79 characters. Do have a look at the other PEP8speaks issues and see if they can be fixed.

Reply:
Done, apart from links to PDB